spec/index.html

```diff
@@ -231,11 +231,16 @@ <h2>Turtle Language</h2>
     <h3>Simple Triples</h3>
     <p>The simplest triple statement is a sequence of
       (<a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a>,
-      <a data-cite="RDF12-CONCEPTS#dfn-predicate">predicate</a>,
+      <a data-cite="RDF12-CONCEPTS#dfn-predicate">predicate</a>, and
       <a data-cite="RDF12-CONCEPTS#dfn-object">object</a>) terms,
-      forming an <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">RDF triple</a>
-      and terminated by <a href="#cp-full-stop"><code title="full stop">.</code></a>.
-      White space (<a href="#cp-space"><code title="space">spaces</code></a>, and/or <a href="#cp-tab"><code title="horizontal tab">tabs</code></a>) may surround terms,
+      forming an <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">RDF triple</a>,
+      terminated by a full stop (<a href="#cp-full-stop"><code title="full stop">.</code></a>).
+      White space
+      (<a href="#cp-space"><code title="space">spaces</code></a>,
+      <a href="#cp-tab"><code title="horizontal tab">tabs</code></a>,
+      <a href="#cp-line-feed"><code title="line feed">LF</code></a>, and/or
+      <a href="#cp-carriage-return"><code title="carriage return">CR</code></a>)
```

Comment:
As I think I noted somewhere else earlier today, I think the `LF` and `CR` should both be pluralized (i.e., `LFs` and `CRs`), just like the `spaces` and `tabs`

Reply:
See #107